pages/style-guide/prose.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Prose Style Guide
+---
+
+## Formatting Conventions
+
+Articles about all content (sites, README, docs, etc) are authored with [GitHub Flavored Markdown](http://github.github.com/github-flavored-markdown/), and the beginning of each article contains some [YAML](http://www.yaml.org/) "front matter" that contains metadata used when the article is published.
+
+### Article Header Metadata
+
+Each article should have the following header (see below as some metatags are optional):
+
+```
+---
+title:  <article title>
+level:  [beginner|intermediate|advance]
+source:  <url of source of the material derived>
+attribution:
+  - Ralph Whitbeck <ralph@email.com>
+  - John Paul <john@email.com>
+---
+```
+
+The `source` and `attribution` properties are optional, and should be used primarily if you are importing an article from an outside source where it was published before being donated for inclusion. You should **not** include these properties if you are adding a new article or editing an existing one, as your contribution will be acknowledged via the git commit logs.
+
```

Comment:
I feel like you're not aware of all the various types of content and sites we have and therefore are having a lot of trouble getting this document written properly.

The `source` and `attribution` properties only exist in the learn.jquery.com repo. They have no meaning anywhere else and therefore don't belong in the style guide.